test/quantization/fx/test_quantize_fx.py

```diff
@@ -5458,7 +5458,7 @@ def forward(self, x):
         b = torch.randn(4)
         m = M(w, b).eval()
         qconfig_dict = {
-            # this has no effect on reshape since it's a CopyNode
+            # reshape will be quantized to fp16 as requested by this qconfig
```

Comment:
@vkuzo I think we can rely on default_reuse_input_config now to configure reshape op to inherit the qconfig from input, by default it can use what is configured for that op, let me know what do you think

Reply:
sorry, not sure what the question is, could you clarify?

Reply:
nvm, I thought I changed the behavior in this PR, but turns out I didn't...
this test case is the one that we have a global fp16 config for reshape, and checks if it has any effect on reshape when it is surrounded by int8 ops. current behavior is that we would respect the qconfig.